settings.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const sourceMapSupport = require('source-map-support')
+const settings = require('libtap/settings');
+
+sourceMapSupport.install({environment:'node', hookRequire: true})
+
+if (+process.env.TAP_DEV_LONGSTACK !== 1) {
```

Comment:
Scrutiny here might be useful.  Could probably add `append-transform` and `caching-transform` to ignoredPackages.